packages/@ember/-internals/views/lib/mixins/action_support.ts

```diff
@@ -5,8 +5,16 @@ import { inspect } from '@ember/-internals/utils';
 import { Mixin, get } from '@ember/-internals/metal';
 import { assert } from '@ember/debug';
 
-const mixinObj = {
-  send(actionName, ...args) {
+/**
+ @class ActionSupport
+ @namespace Ember
+ @private
+*/
+interface ActionSupport {
+  send(actionName: string, ...args: unknown[]): void;
```

Comment:
@chriskrycho's favorite!

Reply:
***BURN IT TO THE GROUND. SALT THE GROUND WHERE IT STOOD. GLASS THE GROUND FROM ORBIT.***